fastestimator/backend/_resize3d.py

```diff
@@ -31,8 +31,9 @@ def resize_3d(tensor: Tensor, output_shape: List[int], resize_mode: str = 'neare
                                                             [[[4.], [4.], [5.], [5.]], [[4.], [4.], [5.], [5.]], [[6.], [6.], [7.], [7.]], [[6.], [6.], [7.], [7.]]],
                                                             [[[4.], [4.], [5.], [5.]], [[4.], [4.], [5.], [5.]], [[6.], [6.], [7.], [7.]], [[6.], [6.], [7.], [7.]]]]]
     ```
-
+    Currently torch only support 32 bit tensors.
```

Comment:
torch.nn.functional.interpolate only allowing 32bit and throwing an error while using 16 bit tensor.